meta-xt-domu/recipes-kernel/linux/linux-virtio-armv8.bb

```diff
@@ -0,0 +1,34 @@
+DESCRIPTION = "Virtio recipe for AArch64 kernels"
+
+inherit logging
+require recipes-kernel/linux/linux-yocto.inc
+
+# Skip processing of this recipe if it is not explicitly specified as the
+# PREFERRED_PROVIDER for virtual/kernel. This avoids errors when trying
+# to build multiple virtual/kernel providers, e.g. as dependency of
+# core-image-rt-sdk, core-image-rt.
+python () {
+    if d.getVar("PREFERRED_PROVIDER_virtual/kernel", True) != "linux-virtio-armv8":
+        raise bb.parse.SkipPackage("Set PREFERRED_PROVIDER_virtual/kernel to linux-virtio-armv8 to enable it")
+}
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+XT_BSP_URL = "git://github.com/torvalds/linux.git"
```

Comment:
It is funny to see how `XT_BSP_URL` points to non Xen-Troops repo. Do you really need this variable?

Reply:
I prefer to use the separate variable for URL, but agree that name could be changed.
I will change it.